### DISCUSSION TITLE
If the first user logging in is non-admin/limited, will the fan control still work?

### DISCUSSION THREAD

**Discussion by IslandJohn** [Rem0o/FanControl.Releases#1625] [General]
It seems like there's a fan control service (although it's not shown as running in Services).

It's not clear (and can't test objectively because you can't launch the UI as non-admin) if the config is loaded by this service, or whether the UI controls the fans.

**Comment by Rem0o** [comment]
There is no seperate service.  It's a single executable running everything, from UI to the fan control part itself.  The program needs to be started with elevated privilege, otherwise it may start, but you won't get any sensor.

**Comment by IslandJohn** [comment]
Ah thanks. Are there any plans to split this it out into an elevated service and UI? According to SysInternals AutoRuns there's a FanControl.sys service registered, but it's not showing up in Services. Is this a work in progress?

**Reply by Rem0o** [reply]
That’s Winring0 running. It is being renamed to the executable’s name to avoid conflicting.  There is some work in progress for a service, but we’re far from there.
